autopilot/autopilot.py

```diff
@@ -126,8 +127,17 @@ def init(configuration, options, plugin):
     plugin.num_channels = int(options['autopilot-num-channels'])
     plugin.percent = int(options['autopilot-percent'])
     plugin.min_capacity_sat = int(options['autopilot-min-channel-size-msat']) / 1000
+    plugin.initialized = threading.Event()
+    plugin.autopilot = None
+    print('Initialized autopilot function')
 
-    plugin.autopilot = CLightning_autopilot(plugin.rpc)
+    def intialize_autopilot():
+        plugin.autopilot = CLightning_autopilot(plugin.rpc)
+        plugin.initialized.set()
+
+    # Load the autopilot in the background and have it notify
+    # dependents once we're finished.
+    threading.Thread(initialize_autopilot, daemon=True).start()
```

Comment:
We need to name the `target` argument, otherwise it thinks that this is `group` which, if set, must be `None`.
```
    threading.Thread(target=initialize_autopilot, daemon=True).start()
```